homeassistant/components/media_player/__init__.py

```diff
@@ -1089,7 +1089,7 @@ def get_browse_image_url(
         """Generate an url for a media browser image."""
         url_path = (
             f"/api/media_player_proxy/{self.entity_id}/browse_media"
-            f"/{media_content_type}/{media_content_id}"
+            f"/{media_content_type}/{quote(quote(media_content_id,safe=''))}"
```

Comment:
Why do we call `quote` twice?

Reply:
Quoting twice safely escapes a forward slash which may be present in the media_content_id. Otherwise, if it's only escaped once, since aiohttp unquotes it before the dynamic path matching, the paths will be matched with the extra forward slashes included which will cause the path matching to break.
Note that in #75105, a `str.replace` was used instead of the first `quote`, which also fixes the path matching. However, that solution is not robust. Consider the case where the original media_content_id is "/%2F". A replace followed by an unquote gives "//", while the above solution correctly gives the original "/%2F".